packages/lesswrong/components/posts/usePostsItem.tsx

```diff
@@ -183,15 +186,23 @@ export const usePostsItem = ({
   const hadUnreadComments =  compareVisitedAndCommentedAt(post.lastVisitedAt, lastCommentedAt);
   const hasNewPromotedComments =  compareVisitedAndCommentedAt(post.lastVisitedAt, lastCommentPromotedAt);
 
-  let postLink = post.draft && !post.debate
-    ? `/editPost?${qs.stringify({postId: post._id, eventForm: post.isEvent})}`
-    : postGetPageUrl(post, false, sequenceId || chapter?.sequenceId);
+  let newPostLink;
```

Comment:
don't like the naming here.  would probably keep this as `postLink` and call the new component prop `overridePostLink` or something.

also nit, but imo better to switch the order of the `if/else` cases